security/certificates/api-server.adoc

```diff
@@ -11,4 +11,9 @@ cluster CA. Clients outside of the cluster will not be able to verify the
 API server's certificate by default. This certificate can be replaced
 by one that is issued by a CA that clients trust.
 
+[NOTE]
+====
+In hosted control plane clusters, self-signed certificates from the API cannot be replaced.
```

Comment:
Maybe consider more active?

```suggestion
In hosted control plane clusters, you cannot replace self-signed certificates from the API.
```

Reply:
Good suggestion! And thanks for the reminder that I merge my own work after it's been peer reviewed 😄